packages/@ember/-internals/glimmer/lib/modifiers/action.ts

```diff
@@ -241,6 +241,20 @@ export default class ActionModifierManager implements ModifierManager<ActionStat
       dom,
       tag
     );
+
+    deprecate(
+      `Using the \`{{action}}\` modifier with \`${actionState.eventName}\` events has been deprecated.`,
+      actionState.eventName !== 'mouseEnter' &&
```

Comment:
I'm a tad worried about:

```hbs
<div {{action 'foo' on=this.mouseMethod}}></div>
```

Do we have a test for that? Does it also emit the deprecation warning?

Reply:
There already is: https://github.com/emberjs/ember.js/pull/18214/files#diff-6f2e4a5d59aa17cdb7b0436b606195b1R1763